packages/loader/container-loader/src/container.ts

```diff
@@ -1470,6 +1470,9 @@ export class Container
 		resolvedUrl: IResolvedUrl,
 		pendingLocalState?: IPendingContainerState,
 	) {
+		const verboseLoggingEnabled =
+			this.mc.config.getBoolean("Fluid.enableVerboseLogging") ?? false;
```

Comment:
The problem you are solving is not unique, and like solutions would be beneficial to be shared.
I'd advice to add verboseness property on events (with some defaults), such that consumers can drop verbose events.
We can use similar idea to TaggedLoggerAdapter where we do that automatically for now (thus verbose events would not go to Loop telemetry unless this same flight is ON). And use such implementation on the boudary of our code and consumer code (in this ragard, it can't be loadRuntime() who uses such wrapper, it should be Loader.constructior, where it calls DebugLogger.mixinDebugLogger - that's the right place to drop any verbose events (but let through through to console).

Reply:
+ @alexvy86  who is looking at similar things for sampling.